src/Samples/Sample.AspNetCore.SystemTests/Test/Base/TestBase.cs

```diff
@@ -1,92 +1,73 @@
-                    .UseBaseUrl("https://YourBaseUrl.com/")
-            .Build();
-            AtataContext.Current.Driver.Maximize();
-#elif RELEASE
-
-            AtataContext.Configure()
-                .UseChrome()
-                .WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions)
-                .UseBaseUrl(Environment.GetEnvironmentVariable("Swedbank.Pay.Sdk.SampleWebsite.BaseUrl", EnvironmentVariableTarget.User))
-                .Build();
-            AtataContext.Current.Driver.Maximize();
-#endif
-        }
-
-        protected TestBase(string driverAlias) => this._driverAlias = driverAlias;
-
-        [TearDown]
-        public void TearDown()
-        {
-            AtataContext.Current?.CleanUp();
-        }
-
-
-        [OneTimeTearDown]
-        public void GlobalDown()
-        {
-            foreach (Driver driverType in Enum.GetValues(typeof(Driver)))
-                WebDriverCleanerService.KillWebDriverProcess(WebDriverCleanerService.DriverNames[driverType]);
-        }
-    }
+﻿using System;
+using Atata;
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.IE;
+
+using Sample.AspNetCore.SystemTests.Services;
+
+namespace Sample.AspNetCore.SystemTests.Test.Base
+{
+    using static Drivers;
+
+    [TestFixture(DriverAliases.Chrome)]
+    [Parallelizable(ParallelScope.All)]
+    public abstract class TestBase
+    {
+        private readonly string _driverAlias;
+
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            AtataContext.GlobalConfiguration.
+                UseChrome().
+                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions).
+                UseFirefox().
+                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Firefox) as FirefoxOptions).
+                UseInternetExplorer().
+                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.InternetExplorer) as InternetExplorerOptions).
+                UseAllNUnitFeatures().
+                UseVerificationTimeout(TimeSpan.FromSeconds(10)).
+                UseElementFindTimeout(TimeSpan.FromSeconds(15)).
+                UseWaitingTimeout(TimeSpan.FromSeconds(15));
+        }
+
+
+        [SetUp]
+        public void SetUp()
+        {
+            var chromeOptions = DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions;
+            AtataContext.Configure()
+                        .UseChrome()
+                        .WithOptions(chromeOptions)
+                        .UseBaseUrl("https://127.0.0.1:5001")
+                        .Build();
+        }
+
+        protected TestBase(string driverAlias) => this._driverAlias = driverAlias;
+
+        [TearDown]
+        public void TearDown()
+        {
+            //if (TestContext.CurrentContext?.Result?.Outcome.Status == NUnit.Framework.Interfaces.TestStatus.Failed)

```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)